docs/measures.md

```diff
@@ -86,7 +86,7 @@ measures = [
 This differs from a normal study definition due to the addition of the `measures` object, which is a list of calls to the `Measure()` function, for each measure. 
 
 #### The `index_date`
-Each month/week start date covered by your study period is passed to the study definition as the `index_date` in turn, thus allowing the cohort and variables to vary each month/week as required. Some variables can retain fixed or no date ranges, e.g. a person's ethnicity would not be expected to vary over time. Read more about using index_date [here](/study-def-dates.md/#index-dates).
+Each month/week start date covered by your study period is passed to the study definition as the `index_date` in turn, thus allowing the cohort and variables to vary each month/week as required. Some variables can retain fixed or no date ranges, e.g. a person's ethnicity would not be expected to vary over time. Read more about using `index_date` [here](/study-def-dates.md/#index-dates).
```

Comment:
nit, and only because it's caught my eye now: If we're tweaking this line anyway, could we rewrite this slightly to remove the use of "here" for the link.

Not essential, but would be nice :sparkles:

See [a previous discussion ~~here~~](https://github.com/opensafely/documentation/pull/372#discussion_r692155882).

Reply:
Nice strike-through 😆 

Thank you for drawing my attention to the link. I'm actually going to remove it, because:
* It's broken. This is probably a formatting issue, because [the target](https://docs.opensafely.org/study-def-dates/#index-dates) still exists. More importantly, though...
* The information contained at the target isn't applicable. You can't supply a date expression when generating measures (i.e. with `--index-date-range`) and the `index_date` in the study definition is overridden. It seems unreasonable to say "See this thing with the same name, that doesn't work in the same way" 🙂